ode_composer/sbl.py

```diff
@@ -98,19 +99,25 @@ def regularizer(self, w):
         return z_sqrt @ w_abs
 
     def objective_fn(self, w):
-        return self.data_fit(w) + self.lambda_param * self.regularizer(w)
+        return self.data_fit(w) + 2*self.lambda_param * self.regularizer(w)
 
     def estimate_model_parameters(self):
         try:
             self.problem.solve(**self.solver_keywords)
 
             if self.config["solver"]["show_time"]:
+<<<<<<< HEAD
```

Comment:
here some text remained when you resolved a merge conflict

Reply:
This should be solved now

Reply:
ok, thanks!